elastic/kibana
# Geo containment alert sparsity handling: preserve active status for non-updated alerts

Carries through active status for geo containment alerts that don't receive updates. To test, likely requires the [faketracks script](https://github.com/thomasneirynck/faketracks) with the feature omitting capabilities added in [this PR](https://github.com/thomasneirynck/faketracks/pull/1).

To test:
1. Set up containment alert w/ faketracks data using the random feature omission setting set to `true`
2. Observe that features that enter shapes but don't receive updates remain active in the alerting dashboard

![image](https://user-images.githubusercontent.com/1728976/101559456-64848100-397e-11eb-8f30-ed68d4299409.png)

_Note: I would recommend using a boundary dataset with greater shape density than the one shown, otherwise you might find yourself waiting a while before a non-updated feature falls within a shape_
